templates/sshd_config.j2

```diff
@@ -6,4 +6,6 @@ Match Group {{ sftp_group_name }}
     AllowTCPForwarding no
     X11Forwarding no
     ForceCommand internal-sftp
+    {% if sftp_allow_passwords %}PasswordAuthentication yes{% endif %}
```

Comment:
@johanmeiring, I'm not sure if this is preferable over something like:
```
PasswordAuthentication {% if sftp_allow_passwords %}yes{% else %}no{% endif %}
```

In other words, should the fact that we default to `False` mean that we explicitly disable password authentication by default?